src/Backoffice/Airline/App/Requests/UpsertAirlineRequest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Airline\App\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Lightit\Backoffice\Airline\Domain\DataTransferObject\AirlineDto;
+use Lightit\Rules\NameUniqueForAirlines;
+
+class UpsertAirlineRequest extends FormRequest
+{
+    public const NAME = 'name';
+
+    public const DESCRIPTION = 'description';
+
+    public function rules(): array
+    {
+        return [
+            self::NAME => [
+                'string',
+                'max:255',
+                new NameUniqueForAirlines(),
```

Comment:
```suggestion
    {
        /** @var Airline|null $airline */
        $airline = $this->route('airline');
    
        return [
            self::NAME => [
                'string',
                'max:255',
                Rule::unique(Airline::class)->ignore($airline),
```
Use Laravel's built-in rule.

Reply:
I agree, but wanted her to see how to create a custom rule, so it's fine as is